docker/base.dockerfile

```diff
@@ -34,6 +34,8 @@ RUN mkdir -p /usr/local/faasm/runtime_root/etc
 RUN cp deploy/conf/hosts /usr/local/faasm/runtime_root/etc/
 RUN cp deploy/conf/resolv.conf /usr/local/faasm/runtime_root/etc/
 RUN cp deploy/conf/passwd /usr/local/faasm/runtime_root/etc/
+RUN mkdir -p /usr/local/faasm/runtime_root/tmp
```

Comment:
This directories were created as part of the [ansible runtime provisioning](https://github.com/faasm/faasm/pull/582/files#diff-f556bb4689b010ab02f0136f191d5b17fc555de156000826d1534c0f32739f42) as well.